agot-bg-website/api/serializers.py

```diff
@@ -24,13 +25,15 @@ class GameSerializer(ModelSerializer):
 
     class Meta:
         model = Game
-        fields = ['id', 'name', 'owner', 'serialized_game', 'view_of_game', 'state', 'version', 'players']
+        fields = ['id', 'name', 'owner', 'serialized_game', 'view_of_game', 'state', 'version', 'players', 'update_last_active']
 
     def update(self, instance, validated_data):
         instance.version = validated_data.pop('version', instance.version)
         instance.serialized_game = validated_data.pop('serialized_game', instance.serialized_game)
         instance.view_of_game = validated_data.pop('view_of_game', instance.view_of_game)
         instance.state = validated_data.pop('state', instance.state)
+        if validated_data.pop('update_last_active', False):
```

Comment:
@Longwelwind So basically I think it must happen here, though I still don't understand how the game servers `patch` call on the websites master api finally lands here...

But if I am correct we have to add a new Property for this because with the databases' `auto_now` feature we won't reach the goal. If so, can you pls provide me the migrations for it like the last time?

Reply:
Django does a lot of the request routing magically and invisibly. What happens is that `GameViewSet` intercepts the `PATCH` request and calls the `update` method of `GameSerializer`.

`auto_now` makes the field auto-updates whenever the entry is updated, for any reason (for example, even if you close a game through the button in the Games page). Doing the way you have done it is a good idea, I think.